templates/deployment.yaml

```diff
@@ -262,14 +278,11 @@ spec:
           protocol: TCP
         - containerPort: 30002
           protocol: TCP
-        volumeMounts:
-          - name: volumes
-            mountPath: "{{ .Values.workbench.volume_path }}"
         env:
-          - name: VOLUME_NAME
-            value: "{{ .Values.workbench.volume_name }}"
-          - name: VOLUME_PATH
-            value: "{{ .Values.workbench.volume_path }}"
+          - name: SHARED_VOLUME_PATH
+            value: "/tmp/shared"
```

Comment:
Is it ok to default this to a `/tmp` folder? Since this will just create an emptyDir if no data is placed there, it's probably fine, but a naive user could accidentally place their ata here not knowing the consequences...

Can we expect that anyone with enough know-how enough to run a Helm chart would know that the `/tmp` directory is cleared when the machine restarts?